### PR TITLE
feat: Display line break in cy.log

### LINKED ISSUE #24964: Cypress.log to display line break
### Current behavior

When logging some text with Cypress.log, the "\n" does not break line for message (nor for displayName), just leaves a space.

```
Cypress.log({
              name: "",
              displayName: "",
              message: "I'm the first line!" + "\n" + "I'm the second line",

```

output
`
<span class="command-message-text">I'm the first line!
I'm the second line</span>
`

image

### Desired behavior

Would be nice that the log view can display the break lines correctly when a string contains "\n"

### Test code to reproduce

```
Cypress.log({
              name: "",
              displayName: "",
              message: "I'm the first line!" + "\n" + "I'm the second line",
            });
```

### Cypress Version

11.2.0

### Node version

16.17.0

### Operating System

13.0.1

### Debug Logs

_No response_

### Other

_No response_

### PR DESCRIPTION
**Pull request by nilgaar**
- Closes #24964 

### User facing changelog
The `cy.log()` `message` will display a line breaks if containing `\n`.
Also, `Cypress.log` `message` will, of course, behave the same way.

### Additional details
This feature enriches the `cypress open` functionality, by adding the line break to the `log` command.
Staking `\n` will be translated in a single line break.

#### `cy.log`
1. Build up a cypress test, add a log like: `cy.log('I am the first line! \n I am the second line!'`
2. Run your test with `cypress open`

#### `Cypress.log`
1. Build up a cypress test, add a log like: `Cypress.log({displayName:'I won\'t be \n broken!!', message:"but I \n will"})`
2. Run your test with `cypress open`

### How has the user experience changed?
Not sure if this can be considered a user experience change but:
from this:
image

to this:
image

### PR Tasks

- [na] Have tests been added/updated?
- [na] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [na] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [na] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

File: packages/reporter/src/commands/command.tsx [hunk 1]
```diff
@@ -40,8 +40,9 @@ export const formattedMessage = (message: string) => {
   const spaceEscapedText = textToConvert.replace(/^ +/gm, (initialSpaces) => '&#32;'.repeat(initialSpaces.length)) // &#32 is the HTML entity for a space
   const converted = md.renderInline(spaceEscapedText)
   const assertion = (split[1] && [`<strong>${split[1].trim()}</strong>`]) || []
+  const withLineBreaks = converted.replace(/\n/g, '<br>')
```
Comment (mike-plummer) [inline packages/reporter/src/commands/command.tsx]: It looks like `markdown-it` has built-in support for this through the [`breaks` parameter](https://markdown-it.github.io/markdown-it/#MarkdownIt.new) (which defaults to false). We should be able to update line 27 to something like this to get the same behavior

```
const md = new Markdown({ breaks: true })
```

**Review by mike-plummer — CHANGES_REQUESTED** [review]

**Review by AtofStryker — CHANGES_REQUESTED** [review]
Hi @nilgaar. Thank you for opening a PR! Nothing here stands out to me that would cause significant issues but are you able to add a test in the driver in [log.cy.ts](https://github.com/cypress-io/cypress/blob/develop/packages/driver/cypress/e2e/cypress/log.cy.js) to verify the line break behavior?

**Comment by AtofStryker** [comment]
@mike-plummer are you able to take another look at this?

**Review by BlueWinds — APPROVED** [review]

**Review by lmiller1990 — APPROVED** [review]

**Comment by lmiller1990** [comment]
Two approvals, let's merge this up. Thanks for the contribution!

**Pull request merged**